examples/basic-app/src/private/BasicCard.tsx

```diff
@@ -0,0 +1,43 @@
+import Box from '@mui/material/Box'
+import Card from '@mui/material/Card'
+import CardActions from '@mui/material/CardActions'
+import CardContent from '@mui/material/CardContent'
+import Button from '@mui/material/Button'
+import Typography from '@mui/material/Typography'
+
+function BasicCard () {
```

Comment:
if this is a custom component of basic-app, use a different name to avoid confusion with the reusable Basic* components of the library

Reply:
Done ✅